### PR TITLE
Middleware monitoring config

### PR DESCRIPTION
**Pull request by sedroche**
## Additional Information
Ticket: https://issues.jboss.org/browse/INTLY-666

This should expose the 3scale metrics to the middleware-monitoring stack using [this method](https://issues.jboss.org/browse/INTLY-718?focusedCommentId=13697604&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13697604).
## Verification Steps

1. Run this installation branch.
2. Go to the Prometheus UI. (`prometheus-route`/config  in the `middleware-monitoring` namespace) and make sure a new config is present.

config

3. Ensure there are 2 new targets at `prometheus-route`/targets
targets
4. Ensure you can run a query v the new targets. e.g. `nginx_http_connections{_3scale_component_element="staging"}` at `prometheus-route`/graph
graph

### CODE REVIEW

**Comment by aidenkeating** [comment]
:eyes:

File: evals/playbooks/install_middleware_monitoring.yml [hunk 1]
```diff
@@ -3,4 +3,5 @@
   gather_facts: no
   roles:
   - role: middleware_monitoring
-    when: middleware_monitoring | default(true) | bool
+  - role: middleware_monitoring_config
+    when: middleware_monitoring | default(true) | bool
```
Comment (aidenkeating) [inline evals/playbooks/install_middleware_monitoring.yml]: I think we need the `when` on both steps, otherwise `middleware_monitoring` will always run

**Comment by aidenkeating** [comment]
@sedroche One small comment, looks good

File: evals/playbooks/install_middleware_monitoring.yml (hunk 1 above)
Comment (sedroche) [reply evals/playbooks/install_middleware_monitoring.yml]: :+1:

**Comment by sedroche** [comment]
Closing as https://github.com/integr8ly/installation/pull/386 includes this work now

**Pull request closed without merging**